packages/docs/components/checkbox.md

```diff
@@ -266,82 +266,81 @@ export default {
-      <input class="ods-checkbox" type="checkbox" name="checkbox-required" id="checkbox-required" value="terms-accepted" checked required>
-      <label class="ods-checkbox--label" for="checkbox-required">I agree to share the donut assortment.</label>
+      <input class="ods-checkbox" type="checkbox" name="example-1" id="example-1" value="terms-accepted" checked required>
+      <label class="ods-checkbox--label" for="example-1">Inline label</label>
     </fieldset>
   </div>
 
   ```html
   <fieldset class="ods-fieldset">
-    <input class="ods-checkbox" type="checkbox" name="checkbox-required" id="checkbox-required" value="terms-accepted" checked required>
-    <label class="ods-checkbox--label" for="checkbox-required">I agree to share the donut assortment.</label>
+    <input class="ods-checkbox" type="checkbox" name="example-1" id="example-1" value="terms-accepted" checked required>
+    <label class="ods-checkbox--label" for="example-1">Inline label</label>
   </fieldset>
   ```
 </figure>
 
-## <span class="u-visually-hidden">Checkbox</span> State: Disabled
+### <span class="u-visually-hidden">Checkbox</span> State: Disabled
 
 <figure class="docs-example">
   <div class="docs-example--rendered">
     <fieldset class="ods-fieldset">
-      <input class="ods-checkbox" type="checkbox" name="checkbox-required" id="checkbox-required" value="terms-accepted" disabled>
-      <label class="ods-checkbox--label" for="checkbox-required">I agree to share the donut assortment.</label>
+      <input class="ods-checkbox" type="checkbox" name="example-2" id="example-2" value="terms-accepted" disabled>
+      <label class="ods-checkbox--label" for="example-2">Inline label</label>
     </fieldset>
   </div>
 
   ```html
   <fieldset class="ods-fieldset">
-    <input class="ods-checkbox" type="checkbox" name="checkbox-required" id="checkbox-required" value="terms-accepted" disabled>
-    <label class="ods-checkbox--label" for="checkbox-required">I agree to share the donut assortment.</label>
+    <input class="ods-checkbox" type="checkbox" name="example-2" id="example-2" value="terms-accepted" disabled>
+    <label class="ods-checkbox--label" for="example-2">Inline label</label>
   </fieldset>
   ```
 </figure>
 
-## <span class="u-visually-hidden">Checkbox</span> State: Error
+### <span class="u-visually-hidden">Checkbox</span> State: Error
 
 <figure class="docs-example">
   <div class="docs-example--rendered">
     <fieldset class="ods-fieldset">
-    <input class="ods-checkbox" data-invalid type="checkbox" name="checkbox-required" id="checkbox-required" value="terms-accepted" required>
-      <label class="ods-checkbox--label" for="checkbox-required">I agree to share the donut assortment.</label>
-      <aside class="ods-field--error" id="checkbox-invalid-error">This field is required.</aside>
+    <input class="ods-checkbox" data-invalid type="checkbox" name="example-3" id="example-3" value="terms-accepted" required>
```

Comment:
We may want to show this example as part of a set paired with a legend, since that's the most common product use case.

Reply:
Good call, fixing.

Reply:
fixed